app/models/loan_question.rb

```diff
@@ -145,7 +96,7 @@ def active?
   end
 
   def child_groups
-    children_sorted_by_position.select(&:group?)
+    children.select(&:group?).sort_by(&:position)
```

Comment:
This method is used in only one place in the memo TOC. Since we're already handling sorting in the Filtered* classes and the TOC is using those (right?), maybe we should move this to those classes? Sorting responsibility has mostly moved there, right? It would probably become just `children.select(&:group?)` since sorting is built into `children`?